form.yml

```diff
@@ -12,9 +12,11 @@ attributes:
     widget: "select"
     label: "Jupyter Lab Container"
     options:
-      - ["scipy-notebook", "scipy-notebook.sif"]
```

Comment:
Do we want to move `scipy-notebook` down in the order? That's the default Jupyter with just Python. I believe the `datascience-notebook` image has R and Julia also, which I've been under the impression that fewer people need. I'm fine with changing this for now, just want to make sure we do it intentionally.

Also, we don't need to do this during semester startup, but it'd be nice to align the way this app works with the `local` folder pattern that we use elsewhere.

Reply:
Good point! I think I moved it for alphabetical purposes, will revert back to usage order!